src/javascript/binary/pages.js

```diff
@@ -30,7 +30,11 @@ var trading_times_init = function() {
                   success: function(html){
                             trading_times.replaceWith(html);
                             trading_times = $("#trading-tabs");
-                            trading_times.tabs();
+			    if (page.language() === 'JA') {	
```

Comment:
@negar-binary Have a look at how it will be done with pure css/sass here:
https://github.com/borisyankov/binary-static/blob/newdesign/src/sass/_access.scss